esmvaltool/recipes/recipe_OceanBGC.yml

```diff
@@ -0,0 +1,132 @@
+# 
+# Project
+# EMBRACE, UKESM
+# 
+# Reference
+# Bakker, D. C. E. et al., Earth Syst. Sci. Data, 6, 69-90,
+# doi:10.5194/essd-6-69-2014 (2014)
+# Bianchi, D. et al., Global Biogeochem. Cy., 26, GB2009,
+# doi:10.1029/2011GB004209 (2012)
+# Jones, S. D. et al., J. Adv. Model Earth Syst., in press (2015).
+# Key, R. M. et al.,Global Biogeochem. Cy., 18, GB4031, doi:10.109/2004GB002247
+# (2004)
+# Roedenbeck, C. et al., Ocean Sci., 9, 193-216, doi:10.5194/os-9-193-2013
+# (2013)
+# Takahashi, T. et al., Mar. Chem., 164, 95-125,
+# doi:10.1016/j.marchem.2014.06.004 (2014)
+# 
+# This recipe is part of the ESMValTool
+###############################################################################
+---
+
+datasets:
+# working datasets
+  - {dataset: CanESM2,       project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 2001,  end_year: 2004}
+#  - {dataset: GISS-E2-H,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: HadGEM2-AO,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: HadCM3,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: CanCM4,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: CSIRO-Mk3-6-0,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+
+#####
+# datasets with Errors (right now):
+
+# Problem with latitude > 90.
+#  - {dataset: HadGEM2-CC,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004} 
+#  - {dataset: HadGEM2-ES,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004} 
+
+  
+###
+# Problem with times
+#  - {dataset: MIROC-ESM,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: MIROC-ESM-CHEM,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+
+# Unstructured grids                 
+#  - {dataset: MPI-ESM-LR,  project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2003}
+#  - {dataset: MPI-ESM-MR,  project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+#  - {dataset: ACCESS1-0,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}  
+#  - {dataset: ACCESS1-3,    project: CMIP5,  mip: Oyr,  exp: historical,  ensemble: r1i1p1,  start_year: 1990,  end_year: 2004}
+
+  
+  
+preprocessors:
+  # --------------------------------------------------
+  # Depth-profile preprocessors
+  # --------------------------------------------------
+  prep_depth_integration_1: # For a 2D map global profile
+    fix_data:     
```

Comment:
Do you need the empty `fix_data` and `fix_metadata` dicts? Fixes should be applied fully automatically by the preprocessor and there should be no need to specify them here.

Reply:
@ledm Can you have a look at this?